src/app/overview/line-chart.js

```diff
@@ -1,18 +1,20 @@
+import * as dates from "../utils/date-util.js";
+
 new Chart(document.getElementById('line-chart'), {
     type: 'line',
     data: {
         // x axis - days of the month
         labels: incrementFill(1, 31),
```

Comment:
move the rest functions to the utils/array-util.js

Reply:
Thanks Nick! It's updated now.